# ComputeRadius.srv
---
float32 radius
int8 typedoor